server/src/test/java/com/cloud/server/ManagementServerImplTest.java

```diff
@@ -124,27 +139,27 @@ public class ManagementServerImplTest {
     UserDataManager userDataManager;
 
     @Spy
+    @InjectMocks
     ManagementServerImpl spy = new ManagementServerImpl();
 
     @Mock
     UserVmDetailsDao userVmDetailsDao;
 
     @Mock
     HostDetailsDao hostDetailsDao;
+
+    @Mock
+    GuestOSCategoryDao guestOSCategoryDao;
+
+    @Mock
+    GuestOSDao guestOSDao;
```

Comment:
```suggestion
    GuestOSCategoryDao _guestOSCategoryDao;

    @Mock
    GuestOSDao _guestOSDao;
```
Use the same name for dao as in the class being tested.

Reply:
done